<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-Q4DJ4GC" name="Default test" tests="1" failures="1" timestamp="8 May 2020 11:13:42 GMT" time="91.559" errors="0">
  <testcase name="login" time="37.849" classname="com.bayer.digitalfarming.frontend.selenium.testcases.AccessAmazon">
    <failure type="java.lang.AssertionError" message="Unable to click on the element: &amp;lt;/br&amp;gt;&amp;lt;font color=red&amp;gt;user :-  name=username&amp;lt;/font&amp;gt;
 ">
      <![CDATA[java.lang.AssertionError: Unable to click on the element: </br><font color=red>user :-  name=username</font>
 
	at org.testng.Assert.fail(Assert.java:94)
	at com.bayer.digitalfarming.frontend.selenium.utils.handlers.ElementHandler.writeText(ElementHandler.java:71)
	at com.bayer.digitalfarming.frontend.selenium.pages.AmazonHomePage.enterUser(AmazonHomePage.java:33)
	at com.bayer.digitalfarming.frontend.selenium.testcases.AccessAmazon.login(AccessAmazon.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:663)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:849)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1157)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1200)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1125)
	at org.testng.TestNG.run(TestNG.java:1033)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- login -->
</testsuite> <!-- Default test -->
